app/src/main/java/com/fmpdroid/moneybox/MainActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.fmpdroid.moneybox;
+
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.view.Gravity;
+import android.view.View;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+public class MainActivity extends AppCompatActivity {
+
+    private DrawerLayout drawer;
+    private static String PREFERENCE = "moneybox_preference";
+    private static String TEMP_CHECK = "moneybox_isEmpty";
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        SharedPreferences preferences = getApplicationContext().getSharedPreferences(PREFERENCE, 0);
+        boolean isEmpty = true;//preferences.getBoolean(TEMP_CHECK, true);
+        if (isEmpty){
+            Intent myIntent = new Intent(MainActivity.this, EmptyActivity.class);
+            MainActivity.this.startActivity(myIntent);
+        }else{
+            Toolbar toolbar = findViewById(R.id.toolbar);
+            setSupportActionBar(toolbar);
+            getSupportActionBar().setDisplayShowTitleEnabled(true);
+            getSupportActionBar().setTitle("tae");
+
+            drawer = findViewById(R.id.drawer_layout);
+
+            ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this, drawer, toolbar,
+                    R.string.navigation_drawer_open, R.string.navigation_drawer_close);
```

Comment:
I think the max limit per line has not been reached yet. No need to newline in here